frontend/public/module/k8s/installed-operators.ts

```diff
@@ -0,0 +1,6 @@
+import * as _ from 'lodash-es';
+
+import { K8sResourceKind } from '../../module/k8s';
+
+export const installedOperatorDisplayName = (installedOperator: K8sResourceKind): string =>
+  _.get(installedOperator, 'spec.displayName') || _.get(installedOperator, 'metadata.name');
```

Comment:
Prefer optional chaining

```suggestion
  installedOperator?.spec?.displayName || installedOperator?.metadata?.name;
```